apps/protocol-frontend/src/components/VerificationHover.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Tooltip } from '@chakra-ui/react';
+import React from 'react';
+
+interface VerificationHoverProps {
+  status: 'Verified' | 'Unverified' | 'noFramework';
+  threshold?: number | null;
+  children: React.ReactNode;
+}
+
+const VerificationHover = ({
+  status,
+  threshold,
+  children,
+}: VerificationHoverProps) => {
+  const verificationPropsMap = {
+    Verified: {
+      label:
+        'Feel free to attest to this contribution! Even though it has been verified for this DAO, additional attestations is good. ',
+      variant: 'tertiary',
+    },
+    Unverified: {
+      label:
+        threshold === 1
+          ? `This contribution only needs ${threshold} more attestation to be verified by your DAO! Be the one to push it through!`
+          : `This contribution needs ${threshold} more attestations to be verified by your DAO.`,
+      variant: threshold === 1 ? 'secondary' : 'primaryPurple',
```

Comment:
For some of these it may be good to abstract these different scenarios into configuration objects and have a top level conditional rather than having conditional for each param

Reply:
I made some changes to make the logic more readable but haven't moved it up into a configuration object yet -- wanted to make sure that the cases are all being covered.

I added the state we talked through earlier where a VF is set but attestation_threshold is null